src/styles/blocks/cards.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: $white;
+  border: 1px solid $border;
+  border-radius: 5px;
+  
+  &:nth-last-child(-n + 4) {
+    margin-top: 46px;
+  }
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
+  &__image-container {
+    margin-bottom: 40px;
```

Comment:
To avoiad margin collapsing better is use one way margin across page - top or bottom - not both of them